apps/fluent-tester/src/FluentTester/TestComponents/FocusZone/FocusZoneTest.tsx

```diff
@@ -1,129 +1,148 @@
-            <Button content="#5" style={focusZoneTestStyles.focusZoneButton} componentRef={buttonRefs[5]} onClick={() => onClick(5)} />
-            <Button content="#6" style={focusZoneTestStyles.focusZoneButton} componentRef={buttonRefs[6]} onClick={() => onClick(6)} />
+    <React.Fragment>
+      {[...Array(props.gridHeight)].map((_value, heightIndex: number) => {
+        return (
+          <View key={heightIndex} style={focusZoneTestStyles.focusZoneViewStyle}>
+            {[...Array(props.gridWidth)].map((_value, widthIndex: number) => {
+              const gridIndex = heightIndex * props.gridWidth + widthIndex + 1;
+              return (
+                <GridButton
+                  key={widthIndex}
+                  style={focusZoneTestStyles.focusZoneButton}
+                  componentRef={props?.buttonRefs?.[gridIndex]}
+                  onClick={() => {
+                    props?.onClick?.(gridIndex);
+                  }}
+                >
+                  <Text>{gridIndex}</Text>
+                </GridButton>
+              );
+            })}
           </View>
-          <View style={focusZoneTestStyles.focusZoneViewStyle}>
-            <Button content="#7" style={focusZoneTestStyles.focusZoneButton} componentRef={buttonRefs[7]} onClick={() => onClick(7)} />
-            <Button content="#8" style={focusZoneTestStyles.focusZoneButton} componentRef={buttonRefs[8]} onClick={() => onClick(8)} />
-            <Button content="#9" style={focusZoneTestStyles.focusZoneButton} componentRef={buttonRefs[9]} onClick={() => onClick(9)} />
+        );
+      })}
+    </React.Fragment>
+  );
+};
+
+const Navigation2DFocusZone: React.FunctionComponent = () => {
+  const [defaultTabbableElementIndex, setDefaultTabbableElementIndex] = React.useState<number | null>(5);
+  const gridWidth = 3,
+    gridHeight = 4;
+
+  const refMemo = React.useMemo(
+    () =>
+      Array(gridWidth * gridHeight)
+        .fill(null)
+        .map(() => React.createRef<View>()),
+    [gridWidth, gridHeight],
+  );
+  const buttonRefs = React.useRef(refMemo);
+  const defaultTextNote = 'The defaultTabbableElement is ' + defaultTabbableElementIndex;
+  return (
+    <React.Fragment>
+      <Text>{defaultTextNote}</Text>
+      <Button onClick={() => setDefaultTabbableElementIndex(null)} style={{ marginTop: 10 }}>
+        Clear defaultTabbableElement
+      </Button>
+      <FocusZoneListWrapper>
+        <FocusZone
+          use2DNavigation={true}
+          defaultTabbableElement={defaultTabbableElementIndex ? buttonRefs.current[defaultTabbableElementIndex] : undefined}
+          isCircularNavigation={true}
+        >
+          <View style={focusZoneTestStyles.focusZoneContainer}>
+            {GridOfButtons({
+              gridWidth: 3,
+              gridHeight: 4,
+              buttonRefs: buttonRefs.current,
+              onClick: (index: number) => setDefaultTabbableElementIndex(index),
```

Comment:
Is this setting the currently selected button to the DefaultTabbableElement? So if you tab out/in, focus lands on the last focused button?

Reply:
on the last pressed button -- if you keynav to ```5```, activate w/ space, keynav to ```6```, tab to next control and shift+tab back to the FocusZone, focus will be on ```5```